### PR TITLE
chore: add Node.js 10

### PR DESCRIPTION
**Pull request by DanielRuf**
Node.js 10 is the current stable version and we should test against it too.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -60,20 +59,19 @@ modules.forEach(module => {
   });
 });
 
-gulp.task('build', function(cb) {
-  gulpSequence('common', modules.filter(module => module !== 'common'), cb);
-});
+gulp.task('common', gulp.series(modules
```
Comment (wbhob) [inline gulpfile.js]: why was this changed (what is the significance of changing the name)

Comment (DanielRuf) [reply gulpfile.js]: Look at line 72, there is still `build`.

Comment (wbhob) [reply gulpfile.js]: that seems a bit redundant imo, but I don't know if both scripts are depended on. I think they should be consolidated since one is just a pointer to the other (but like I said, I don't know if there are other tasks depending on each one independently)

**Comment by wbhob** [comment]
any update on this PR?

**Comment by DanielRuf** [comment]
> any update on this PR?

Nothing new from my side, I see no need for further optimization of this PR, my PR is ready for merge so far.
